Cargo.toml

```diff
@@ -1,13 +1,9 @@
 [package]
-name = "{{project-name}}"
+name = "template"
```

Comment:
This repository is meant to be used with [cargo-generate](https://github.com/cargo-generate/cargo-generate), having `package.name` and `package.authors` (removing the template variables) defeats this purpose.